libcloud/dns/drivers/powerdns.py

```diff
@@ -439,13 +485,27 @@ def _to_zones(self, items):
             zones.append(self._to_zone(item))
         return zones
 
-    def _to_record(self, item, zone):
-        return Record(id=None, name=item['name'], data=item['content'],
+    def _to_record(self, item, zone, record = None):
+        if record is None:
+            data = item['content']
+        else:
+            data = record['content']
+        return Record(id=None, name=item['name'], data=data,
                       type=item['type'], zone=zone, driver=self,
                       ttl=item['ttl'])
 
     def _to_records(self, items, zone):
         records = []
-        for item in items.object['records']:
-            records.append(self._to_record(item, zone))
+        if self._pdns_version() == 3:
+            for item in items.object['records']:
+                records.append(self._to_record(item, zone))
+        if self._pdns_version() == 4:
+            for item in items.object['rrsets']:
+                for record in item['records']:
+                    records.append(self._to_record(item, zone, record))
         return records
+		
+    def _pdns_version(self):
+        if self.api_root == '': return 3
```

Comment:
How is version currently declared? To which version does the driver currently default to?

Reply:
EDIT: I see it's declared here - https://github.com/apache/libcloud/blob/9cab4689ac005fed2de0761174cf13981d525128/libcloud/dns/drivers/powerdns.py#L133